backend/src/test/java/capstone/backend/controller/OrderToSupplierControllerTest.java

```diff
@@ -124,12 +123,12 @@ void createOrder() {
     }
 
     @Test
-    void createOrderSavesOrderItem(){
+    void createOrderSavesOrderItem() {
```

Comment:
You could also add a check via GET request if the orderItem was actually added

Reply:
i'm doing it with this line
```  assertThat(orderRepo.findById(order.getId()).get(), is(mapOrder(order)));```